music21/scale/__init__.py

```diff
@@ -847,13 +847,16 @@ class AbstractHarmonicMinorScale(AbstractScale):
     A true bi-directional scale that with the augmented
     second to a leading tone.
 
-    This is the only scale to use the "_alteredDegrees" property.
+    The harmonic and jazz minor scales are the only to
+    use the "_alteredDegrees" property.
     '''
 
     def __init__(self, mode=None):
         super().__init__()
         self.type = 'Abstract Harmonic Minor'
         self.octaveDuplicating = True
+        self.relativeMinorDegree = 1
+        self.relativeMajorDegree = 3
```

Comment:
Likewise, can you explain why this is needed?

Reply:
Without this code, an error message like this is given when using the getRelativeMinor/getRelativeMajor functions on the harmonic/melodic/jazz minor scales:

`AttributeError: 'AbstractHarmonicMinorScale' object has no attribute 'relativeMinorDegree'`

Reply:
Ah, got it, any of the non-natural minors, thanks. Yes, this should work because it's a method on `DiatonicScale`.

Can we move these attribute assignments to `buildNetwork()`? That's where they seem to be in all other cases.